skimage/feature/peak.py

```diff
@@ -11,8 +11,8 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
     Peaks are the local maxima in a region of `2 * min_distance + 1`
     (i.e. peaks are separated by at least `min_distance`).
 
-    If peaks are flat (i.e. multiple adjacent pixels have identical
-    intensities), the coordinates of all such pixels are returned.
+    If multiple peaks in the specified region have identical intensities,
+    the coordinates of all such pixels are returned.
```

Comment:
@jni How do you feel about function returning labeled map (different integer values for different peaks/levels) instead of current binary one. This way it addresses the major anticipated use case (initial seeding for segmentation) and doesn't require significant modification to the code.
If you have other thoughts about `peaks...` usage, it'd be really helpful to know about them.